src/backend/opencl/Array.cpp

```diff
@@ -317,11 +323,19 @@ kJITHeuristics passesJitHeuristics(Node *root_node) {
         constexpr size_t max_nvidia_param_size = (4096 - base_param_size);
         constexpr size_t max_amd_param_size    = (3520 - base_param_size);
 
+        // This value is really for the Intel HD Graphics platform. The CPU
+        // platform seems like it can handle unlimited parameters but the
+        // compile times become very large.
+        constexpr size_t max_intel_cpu_param_size =
```

Comment:
incorrect var name - can lead to confusion `max_intel_igpu_param_size` seems more appropriate

Reply:
You are right it should be:
```suggestion
        constexpr size_t max_intel_igpu_param_size =
```